### PR TITLE
[5.x] Implement translations for dictionary keys used in titles shown in breadcrumb trails

### PR DESCRIPTION
**Pull request by peimn**

### CODE REVIEW

File: src/CP/Breadcrumbs.php [hunk 1]
```diff
@@ -37,7 +37,8 @@ public function jsonSerialize()
 
     public function title($title = null)
     {
-        $crumbs = $this->crumbs->map->text;
+        $new_crumbs = $this->crumbs->map(fn ($v) => ['text' => __($v['text']), 'url' => $v['url']]);
```
Comment (ryanmitchell) [inline src/CP/Breadcrumbs.php:40]: maybe simplify to `$crumbs = $this->crumbs->map(fn ($v) => __($v['text']))` ?

**Comment by jasonvarga** [comment]
What page is this an issue on?

**Comment by peimn** [comment]
> What page is this an issue on?

This title is automatically generated for all pages and displays the path. For example, if a page is opened from a collection, the title of the page will be `Statamic > collections > pages > Main Page`.

If a key, such as `dictionary.pages`, is used for the title, it will appear in the breadcrumb trail exactly as it is.

This change will translate the title.

File: src/CP/Breadcrumbs.php (hunk 1 above)
Comment (peimn) [reply src/CP/Breadcrumbs.php:40]: Nice!

**Comment by jasonvarga** [comment]
But where are you seeing this long breadcrumb? On an entry, it's just `< Collection name`

CleanShot 2024-05-10 at 16 06 31

**Comment by jasonvarga** [comment]
Ah the `<title>` duh! 😆

**Review by jasonvarga — APPROVED** [review]

**Pull request merged**